changelogs/unreleased/gh-9723-sync-system-spaces.md

```diff
@@ -0,0 +1,9 @@
+## feature/replication
+
+* Made all of the system spaces synchronous by default when the synchronous
+  queue is claimed. A `box_consider_system_spaces_synchronous` backward
+  compatibility option was added to control this behaviour. A new read-only
+  `state` subtable for the Lua space object returned from the `box.space`
+  registry with an `is_sync` field was added to reflect the effective state of
+  replication for system spaces. For user spaces, this field will always mirror
+  the is_sync option set for the space (gh-9723).
```

Comment:
* Made all of the system spaces synchronous by default when the synchronous
  queue is claimed. Added a `box_consider_system_spaces_synchronous` backward
  compatibility option to control this behavior. Added a new read-only
  `state` subtable for the Lua space object returned from the `box.space`
  registry. This subtable has an `is_sync` field that reflects the effective
  state of replication for system spaces. For user spaces, this field will
  always mirror the `is_sync` option set for the space (gh-9723).

Reply:
Fixed.